Orange/preprocess/preprocess.py

```diff
@@ -34,12 +34,20 @@ def __call__(self, data):
 
 
 class Continuize(Preprocess):
-    (Indicators, FirstAsBase, FrequentAsBase, Remove, RemoveMultinomial,
-     ReportError, AsOrdinal, AsNormalizedOrdinal, Leave) = Enum(
-         "Continuize",
-         "Indicators, FirstAsBase, FrequentAsBase,"
-         "Remove, RemoveMultinomial, ReportError, AsOrdinal,"
-         "AsNormalizedOrdinal, Leave")
+    class ContinuizeDV(Enum):
```

Comment:
Before, you'd get:
```py
>>> from Orange.preprocess import Continuize
>>> preproc = Continuize(multinomial_treatment=Continuize.FrequentAsBase)
>>> preproc
Continuize(multinomial_treatment=Continuize.FrequentAsBase)
```
With this change, it's:
```py
>>> preproc = Continuize(multinomial_treatment=Continuize.FrequentAsBase)
>>> preproc
Continuize(multinomial_treatment=2)
```
Is there some way to keep the customized Enum and still pickle numbers?

Or, as an alternative, sklearn uses expressive strings. I wouldn't that also.